src/mento.ts

```diff
@@ -326,4 +336,111 @@ export class Mento {
     )
     return exchanges[0]
   }
+
+  /**
+   * Returns the Mento exchange for a given exchange id
+   * @param exchangeId the id of the exchange
+   * @returns the exchange with the given id
+   */
+  async getExchangeById(exchangeId: string): Promise<Exchange> {
+    const exchanges = (await this.getExchanges()).filter(
+      (e) => e.id === exchangeId
+    )
+
+    if (exchanges.length === 0) {
+      throw Error(`No exchange found for id ${exchangeId}`)
+    }
+
+    assert(
+      exchanges.length === 1,
+      `More than one exchange found with id ${exchangeId}`
+    )
+    return exchanges[0]
+  }
+
+  /**
+   * Returns whether trading is enabled in the given mode for a given exchange id
+   * @param exchangeId the id of the exchange
+   * @param mode the trading mode
+   * @returns true if trading is enabled in the given mode, false otherwise
+   */
+  async isTradingEnabled(
+    exchangeId: string,
+    mode: TradingMode = TradingMode.BI_DIRECTIONAL
+  ): Promise<boolean> {
+    const exchange = await this.getExchangeById(exchangeId)
+    const biPoolManager = BiPoolManager__factory.connect(
+      exchange.providerAddr,
+      this.signerOrProvider
+    )
+
+    const [breakerBoxAddr, exchangeConfig] = await Promise.all([
+      biPoolManager.breakerBox(),
+      biPoolManager.getPoolExchange(exchangeId),
+    ])
+
+    const breakerBox = IBreakerBox__factory.connect(
+      breakerBoxAddr,
+      this.signerOrProvider
+    )
+    const currentMode = await breakerBox.getRateFeedTradingMode(
+      exchangeConfig.config.referenceRateFeedID
+    )
+
```

Comment:
It will return a value `!= 0`. At the moment anything other than `0` means the same (trading disabled), but in the future it could be that trading is enabled in a certain mode (e.g. only in a certain direction)